src/common/schema.graphql.ts

```diff
@@ -341,7 +341,17 @@ export default gql`
 		tierId: ID!
 	}
 
+	input EventUpdateInput {
+		name: String!
+		startTimestamp: String!
+		duration: Int!
+		description: String!
+		location: String!
+		eventType: String!
+	}
+
 	type Mutation {
+		addOrUpdateEvent(input: EventUpdateInput!): String
```

Comment:
If it doesn't return a string is that an error? If so, we can make it `String!` and then throw an error if `addOrUpdateEvent` failed.

If it can be successful and not return a string, you may resolve this comment.